account/serializers.py

```diff
@@ -152,3 +152,39 @@ def validate(self, data):
         if data["password"] != data["confirm_password"]:
             raise serializers.ValidationError("Passwords do not match")
         return data
+
+
+
+class UpdateAccountSerializer(serializers.ModelSerializer):
```

Comment:
Let's use the existing Serializer for the Account object and write the update method in it.